server/gitlab/api.go

```diff
@@ -198,10 +198,25 @@ func (g *gitlab) GetProjectHooks(user *UserInfo, owner string, repo string) ([]*
 		return nil, err
 	}
 
-	projectPath := fmt.Sprintf("%s/%s", owner, repo)
-	projectHooks, _, err := client.Projects.ListProjectHooks(projectPath, nil)
-	if err != nil {
-		return nil, err
+	opt := &internGitlab.ListProjectHooksOptions{
+		PerPage: 20,
+		Page:    1,
```

Comment:
Page 1 or 0?

Reply:
@hanzei it starts from 1 can refer here https://docs.gitlab.com/ee/api/index.html#pagination
![image](https://user-images.githubusercontent.com/86650698/166116334-b7ddf17c-a9b0-4269-a901-dd410c237247.png)